news/6605.hooks.rst

```diff
@@ -0,0 +1,4 @@
+Add ``IPython`` to the list of packages in the ``PIL`` hook in order to prevent 
```

Comment:
```suggestion
Add ``IPython`` to the list of excluded packages in the ``PIL`` hook in order to prevent 
```
?

Reply:
Ah, yes, thanks. I've slightly changed the wording there as well.